app/src/main/java/org/simlar/service/liblinphone/LinphoneManager.java

```diff
@@ -520,6 +522,30 @@ public void onCallStatsUpdated(@NonNull final Core lc, @NonNull final Call call,
 		}
 	}
 
+	private String getIceStateUiString(final IceState iceState)
+	{
+		if (iceState == null) {
+			return mContext.getString(R.string.linphone_ice_state_none);
+		}
+
+		switch (iceState) {
+		case NotActivated:
+			return mContext.getString(R.string.linphone_ice_state_not_activated);
+		case Failed:
+			return mContext.getString(R.string.linphone_ice_state_failed);
+		case InProgress:
+			return mContext.getString(R.string.linphone_ice_state_in_progress);
+		case HostConnection:
+			return mContext.getString(R.string.linphone_ice_state_host_connection);
+		case ReflexiveConnection:
+			return mContext.getString(R.string.linphone_ice_state_reflexive_connection);
+		case RelayConnection:
+			return mContext.getString(R.string.linphone_ice_state_relay_connection);
+		}
+
+		return mContext.getString(R.string.linphone_ice_state_unknown);
```

Comment:
Most of the time we did:
```suggestion
			return mContext.getString(R.string.linphone_ice_state_unknown);
		}
```